app/src/main/res/layout/activity_home.xml

```diff
@@ -4,152 +4,226 @@
 
     <ImageView
-        android:id="@+id/imageView4"
-        android:layout_width="74dp"
+        android:id="@+id/folder3"
+        android:layout_width="100dp"
         android:layout_height="60dp"
-        android:layout_marginTop="40dp"
-        app:layout_constraintTop_toBottomOf="@+id/textView3"
-        app:srcCompat="@drawable/folder_bg3"
-        tools:layout_editor_absoluteX="228dp" />
+        android:layout_marginStart="96dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="109dp"
+        android:layout_marginBottom="8dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.076"
+        app:layout_constraintStart_toEndOf="@+id/folder_title_text"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.227"
+        app:srcCompat="@drawable/folder_bg3" />
 
     <Spinner
-        android:id="@+id/spinner2"
+        android:id="@+id/folder_dropdown"
         android:layout_width="91dp"
         android:layout_height="32dp"
-        android:layout_marginTop="16dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="30dp"
         android:layout_marginBottom="8dp"
-        app:layout_constraintBottom_toBottomOf="@+id/view3"
-        app:layout_constraintTop_toBottomOf="@+id/view2"
-        tools:layout_editor_absoluteX="278dp" />
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="@+id/view3"
+        app:layout_constraintHorizontal_bias="0.9"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.354" />
 
     <TextView
-        android:id="@+id/textView4"
+        android:id="@+id/see_more_text"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="4dp"
-        android:layout_marginEnd="124dp"
-        android:layout_marginBottom="8dp"
+        android:layout_marginStart="18dp"
+        android:layout_marginTop="265dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginBottom="3dp"
         android:text="See More"
-        android:textSize="32sp"
-        app:layout_constraintBottom_toBottomOf="@+id/view3"
-        app:layout_constraintEnd_toStartOf="@+id/spinner2"
-        app:layout_constraintTop_toBottomOf="@+id/view2" />
+        android:textSize="23sp"
+        app:layout_constraintBottom_toTopOf="@+id/home_title_dropdown"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.01"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.163" />
 
     <android.support.design.widget.FloatingActionButton
         android:id="@+id/floatingActionButton2"
```

Comment:
what does this button do? the id doesnt tell me anything
also it looks weird idk if intentional- theres also some white around the edges that I can see did you use paint or something to rm the bg
![image](https://user-images.githubusercontent.com/13546886/47969119-cba90c00-e027-11e8-97b0-86d5724bd56b.png)

Reply:
this button was downloaded straight from duke's google drive and uploaded.